bedrock/security/templates/security/bug-bounty/hall-of-fame.html

```diff
@@ -14,7 +14,7 @@ <h1 class="mzp-c-article-title">Firefox Bug Bounty Rewards</h1>
   </header>
 
   <p>On behalf of the Mozilla and the millions of people who visit our sites, use Firefox and our other products we would like to thank them for their hard work in helping to make us more secure.</p>
-  <p>As of this date, we have paid out almost <b>$1,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
+  <p>As of this date, we have paid over <b>$2,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
```

Comment:
Based on the comment from @dveditz it sounds like this should increase even more?

 ```suggestion
  <p>As of this date, we have paid over <b>$4,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
```